### ISSUE TITLE
v0.1.4 broke custom messages

### ISSUE THREAD

**Issue by donncha** [hallelujah/valid_email#120]
Currently producing validation messages like "Customer Email {:message=>\"is invalid\"}".

Specifically, in EmailValidator
```
record.errors.add attribute, message: (msg % {value: value})
```

should revert to
```
record.errors.add attribute, (msg % {value: value})
```

**Comment by oleksiivykhor** [comment]
Yes. Previously I was getting the errors in the same as rails format `record.errors[:email] # => ['is invalid']`, but now it's `record.errors[:email] # => [{:message=>' is invalid'}]`

**Comment by stevendaniels** [comment]
It looks like this behavior was changed in https://github.com/hallelujah/valid_email/pull/118

**Comment by arturoherrero** [comment]
@hallelujah Shall we close this issue after https://github.com/hallelujah/valid_email/pull/125 was merged?

**Comment by jsocol** [comment]
I do not believe #125 would fix this issue. The `errors.add` API changed in Rails 6.1 to accept a hash with a message key as the second argument. [Prior to 6.1](https://guides.rubyonrails.org/v6.0/active_record_validations.html#errors-add), the second argument was expected to be the message itself. Later versions of Rails/ActiveModel still work with the old version of the API, though as #118 demonstrates, there's a good reason to use the new API.

**Comment by hallelujah** [comment]
Shall we bump major version then? Fixing the issue in v0.2.1 which will be the last supporting Rails `<= 6.0`
